ember_debug/object-inspector.js

```diff
@@ -528,6 +528,13 @@ export default EmberObject.extend(PortMixin, {
    */
   mixinDetailsForObject(object) {
     const mixins = [];
+    if (object instanceof Ember.ObjectProxy && object.content && !object._showProxyDetails) {
+      return this.mixinDetailsForObject(object.content);
+    }
+
+    if (object instanceof Ember.ArrayProxy && object.content && !object._showProxyDetails) {
+      return this.mixinDetailsForObject(object.slice(0, 101));
```

Comment:
What is this `slice` for?

Reply:
it does the same as `toArray()`, so it obtains the correct items from e.g. ember data many-array but only for the first 101 items, (though this should actually be part of the perf PR). otherwise it would call `objectAt` for all items in many-array